src/alfasim_score/converter/alfacase/converter_main.py

```diff
@@ -0,0 +1,28 @@
+from alfasim_sdk import CaseDescription
+from alfasim_sdk import convert_description_to_alfacase
+from pathlib import Path
+
+from alfasim_score.common import OperationType
+from alfasim_score.converter.alfacase.base_operation import BaseOperationBuilder
+from alfasim_score.converter.alfacase.injection_operation import InjectionOperationBuilder
+from alfasim_score.converter.alfacase.production_operation import ProductionOperationBuilder
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+
+
+def convert_score_to_alfacase_description(score_filepath: Path) -> CaseDescription:
+    """Convert SCORE input file to an alfacase description."""
+    score_input = ScoreInputReader(score_filepath)
+    operation_type = score_input.read_operation_type()
+    builder: BaseOperationBuilder
+    if operation_type == OperationType.PRODUCTION:
+        builder = ProductionOperationBuilder(score_filepath)
+    elif operation_type == OperationType.INJECTION:
+        builder = InjectionOperationBuilder(score_filepath)
```

Comment:
Include a `else` or it will crash in case there is no match.

Reply:
Maybe this is not needed because the reader breaks if anything different than these two available options are set as operation type in the SCORE file. It breaks with message `ValueError: 'UNKOWN_OPERATION' is not a valid OperationType` in the `read_operation_type` method.

Reply:
Maybe it is better to avoid this crash using `try` `catch`.
And then, if there are only two options for `operation_type` you could use `else`, instead of `elif`.

Reply:
I think this must break the converter if there is a invalid operation name in the parsed file. I used this `elif` approach in this kind of situation because I like to think this like a match-case block and I think (not in this specific situation) sometimes `else` can match all other options and can cause problems when new options ("new cases") are added.

Reply:
If there are only two options, `if` `else` is enough.
If there may be another cases, maybe a match-case block is better.

Reply:
Ok, if-else now with a comment indicating about the other option. 👍 
I'm using now this overwrite to print all the available options, analogous to what Barril does printing all available units when your try to use an invalid one.

```
    @classmethod
    def _missing_(cls, value):
        raise ValueError(
        "%r is not a valid %s.  Valid types: %s" % (
            value,
            cls.__name__,
            ', '.join([repr(m.value) for m in cls]),
        ))
```

Reply:
Good idea

Reply:
Nice!